chart/apisix/templates/ingress.yaml

```diff
@@ -25,6 +25,7 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: {{ $fullName }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
`Ingress` required cluster install `ingress-controller`, k8s cluster does not install any implement about it by default, so this CRD will not work.

Reply:
We can remove this first, and add it back when apisix-ingress-controller is ready.